static/admin/config.yml

```diff
@@ -183,4 +183,18 @@ collections:
               - label: "Color Description"
                 name: "description"
                 widget: "string"
-      - {label: "Version No.", name: "version", widget: "select", options: ["1.01", "1.02", "1.03","1.04", "1.05"]}
\ No newline at end of file
+      - {label: "Version No.", name: "version", widget: "select", options: ["1.01", "1.02", "1.03","1.04", "1.05"]}
+  - name: "versions"
+    label: "Versions"
+    folder: "src/pages/versions"
+    create: false
+    extension: "json"
+    format: "json"
+    editor:
+      preview: false
+    slug: "{{slug}}"
+    identifier_field: srcTemplateKey
+    fields:
+    - {label: "Template Key", name: "templateKey", widget: "hidden", default: version-config"}
```

Comment:
missing `"` on default value. 
Probably the cause of the extra `\"` in the json files

Reply:
oh, will update, thanks